src/node_api_helpers.h

```diff
@@ -38,6 +45,289 @@
   NAPI_DISALLOW_MOVE(CLASS)
 
 namespace Napi {
+  inline napi_value Value() {
+    napi_env env = napi_get_current_env();
+    return napi_new_value(env);
+  }
+
+
+  // Napi::New helpers
+  inline napi_value New() {
```

Comment:
I can see how all these helpers are useful. [I had been thinking](https://github.com/nodejs/abi-stable-node/issues/53) eventually we might create a C++ wrapper class around `napi_value`, that would have a lot of these sort of helper methods (and more). I guess we can go with this for now, and replace with something richer later.